src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -626,8 +623,9 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
             case Success(n) =>
               logger.debug(s"POST /orgs/$orgId/changes node/agbot heartbeat result: $n")
               if (n > 0) {
-                if(qResp.nonEmpty) (HttpCode.POST_OK, buildResourceChangesResponse(qResp, reqBody.maxRecords, reqBody.changeId, maxChangeId))
-                else (HttpCode.POST_OK, ResourceChangesRespObject(List[ChangeEntry](), maxChangeId, maxChangeId, ExchangeApi.adminVersion()))
+                val hitMaxRecords = (qResp.size == reqBody.maxRecords) // if they are equal then we hit maxRecords
```

Comment:
This is a nit, but i like to practice defensive programming: i would make the operator `>=` because even tho we think we know `qResp.size` can never be bigger than `reqBody.maxRecords`, just in case there was a corner case in which it could be, this comparison would still do what we want. The general principle is: any time you can make the code right here less dependent on code somewhere else doing something special, that's a good thing.

But you don't need to change this now, i'm going to approve and merge it as is.